btk/metrics.py

```diff
@@ -104,7 +104,21 @@ def meas_ksb_ellipticity(image, additional_params):
     return result
 
 
-def get_detection_match(true_table, detected_table):
+def distance_center(true_gal, detected_gal):
+    """Computes distance between the two galaxies given as arguments.
+
+    Args:
+        true_gal (astropy.table.Table): Contains information related to the true galaxy.
+        detected_gal (astropy.table.Table): Contains information related to the detected galaxy
+    Returns:
+        Distance between the two galaxies
+    """
+    return np.hypot(
+        true_gal["x_peak"] - detected_gal["x_peak"], true_gal["y_peak"] - detected_gal["y_peak"]
+    )
+
+
+def get_detection_match(true_table, detected_table, f_distance=None):
```

Comment:
why not use `f_distance=distance_center` in the keyword argument here and avoid the `if/else` statement below?

Reply:
I guess the original method of doing it will be faster, although the for loop should always be small even with blends of ~8 unless the detection algorithm is really bad. Not sure if it's worth keeping the original method for sake of speed

Reply:
That's a good point, I will change it.